cdap-docs/developers-manual/source/index.rst

```diff
@@ -57,12 +57,19 @@ CDAP Developers’ Manual
   a daemon to tail local files and an Apache Flume Sink implementation.
 
 
+.. |data-exploration| replace:: **Data Exploration:**
+.. _data-exploration: data-exploration/index.html
+
+- |data-exploration|_ Data in CDAP can be explored without writing any code through the use of ad-hoc SQL-like queries.
```

Comment:
Data in CDAP can be **explored without writing any code** through the use of **ad-hoc SQL-like queries**.